src/nuget/Microsoft.DotNet.Build.Tasks.VersionTools.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.Build.Tasks.VersionTools</id>
+    <version>1.0.0-prerelease</version>
```

Comment:
Yeah, about versioning: I do think all BuildTools packages should be on the same version path for reasonable branching. So here, `2.0.0-servicing`. Some discussion here: https://github.com/dotnet/buildtools/issues/1489. (So don't worry about updating everything here, just this new package.)

Reply:
I updated the prerelease label.  I'll update the version to be 2.0.0 and ignore the other packages.

Reply:
Thanks!